internal/pkg/config/config.go

```diff
@@ -10,6 +10,7 @@ import (
 	"github.com/elastic/go-ucfg"
 	"github.com/elastic/go-ucfg/flag"
 	"github.com/elastic/go-ucfg/yaml"
+	"github.com/rs/zerolog/log"
```

Comment:
I will need to dig into how the logger works and is configured in the fleet-server, something looks weird to me with the `zerolog/ecszerolog` and the `logger package`.

Reply:
the `zerolog/ecszerolog` package sets up the `zerolog` globals necessary to adhere to the ecs format, so as long as it's added before/alongside the logger then it should be "functional". ideally, i would like to move more toward a dependency injection pattern, where we pass things down through either the context/parameters/data members.

Reply:
i think punting on the logging updates makes sense for now, since adjusting the bootstrapping process seems out of scope for this change set. WDYT?